Dockerfile

```diff
@@ -12,7 +15,7 @@ COPY source ./source
 RUN go get -v ./... && \
     go get -u github.com/fsouza/fake-gcs-server/fakestorage && \
     go get -u github.com/kshvakov/clickhouse && \
-    GOOS=linux GOARCH=386 go build -a -o build/migrate.linux-386 -ldflags='-X main.Version=$(VERSION)' -tags '$(DATABASE) $(SOURCE)' ./cli
+    GOOS=linux GOARCH=386 go build -a -o build/migrate.linux-386 -ldflags="-X main.Version=${VERSION}" -tags "$DEPS" ./cli
```

Comment:
Drop `GOARCH` and `GOOS` since we're running inside docker and the binary doesn't need to be portable.

Also, specify the sources.

Reply:
Let's keep the db and source dependencies separate for easier management

Reply:
Done